aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -96,6 +102,15 @@
   public static final String GLUE_CATALOG_SKIP_ARCHIVE = "glue.skip-archive";
   public static final boolean GLUE_CATALOG_SKIP_ARCHIVE_DEFAULT = false;
 
+  /**
+   * If set, GlueCatalog will use Lake Formation for access control.
+   * For more credential vending details, see: https://docs.aws.amazon.com/lake-formation/latest/dg/api-overview.html
+   * If enabled, the {@link AwsClientFactory} implementation must be {@link LakeFormationAwsClientFactory}
```

Comment:
Javadoc does not break newline automatically, similar to the markdown file. So you need to add a dot at the end of L107 to mark the end of the sentence.

Reply:
added. thanks